app/src/main/java/com/marcossalto/targetmvd/models/TopicModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.marcossalto.targetmvd.models
+
+data class TopicModel(
+    val id: Int,
+    val label: TopicTypes,
+    val icon: String = ""
+)
+
+enum class TopicTypes {
+    CHESS,
+    FOOTBALL,
+    TRAVEL,
+    POLITICS,
+    ART,
+    DATING,
+    MUSIC,
+    MOVIES,
+    SERIES,
+    FOOD,
+    MATE
+}
```

Comment:
I think that these topics are coming from the backend.
Check the API reference.

Reply:
Thanks for your suggestion.
The original purpose of this class is to use it as a data model to map each topic image to its corresponding topic. Topic images are stored as resources.